Modules/System Tests/Cryptography Management/src/SignatureKeyTest.al

```diff
@@ -0,0 +1,111 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+codeunit 132590 "Signature Key Test"
+{
+    Subtype = Test;
+
+    var
+        SignatureKey: Record "Signature Key" temporary;
+        LibraryAssert: Codeunit "Library Assert";
+
+    [Test]
+    procedure VerifySignatureKeyIsInitialized()
+    var
+        CertBase64Value: Text;
+    begin
+        // [SCENARIO] Init Signature Key record from Base64 value
+        // [GIVEN] Get Test Certificate Base64
+        CertBase64Value := GetCertificateBase64();
+
+        // [WHEN] Initialize record from Base64 value 
+        SignatureKey.FromBase64String(CertBase64Value, 'Test', true);
+
+        // [THEN] Verify that "Key Value Blob" BLOB has Value
+        LibraryAssert.IsTrue(SignatureKey."Key Value Blob".HasValue, 'Failed to verify certificate.');
```

Comment:
This could be an issue as due to protection level we will not be able to access SignatureKey."Key Value Blob"